client/state/sites/test/selectors.js

```diff
@@ -1184,4 +1194,675 @@ describe( 'selectors', () => {
 			expect( frontPageType ).to.eql( 'page' );
 		} );
 	} );
+
+	describe( '#canJetpackSiteManage()', () => {
+		it( 'it should return `null` for a non-existing site', () => {
+			const state = {
```

Comment:
There is a bit of code duplication around initial state. What do you think about creating helper method which would create basic structure. Something like:

```js
const createState = ( items = {} ) => deepFreeze( {
    sites: {
        items
    }
} );
```

I also think it would be nice to add `deepFreeze` to make sure we never mutate state. Check other tests for reference: https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=deepFreeze.